tests/Selenium2Config.php

```diff
@@ -16,7 +16,7 @@ public static function getInstance()
      */
     public function createDriver()
     {
-        $browser = $_SERVER['WEB_FIXTURES_BROWSER'];
+        $browser = getenv('WEB_FIXTURES_BROWSER') ?: 'firefox';
```

Comment:
Why you're getting `WEB_FIXTURES_BROWSER` via `getenv`, but `DRIVER_URL` via `$_SERVER`?